src/Tools/ExternalAccess/Razor/RazorAnalyzerAssemblyResolver.cs

```diff
@@ -61,7 +61,7 @@ internal sealed class RazorAnalyzerAssemblyResolver() : IAnalyzerAssemblyResolve
             // or the ServiceHubCore subfolder (razor). In the root directory these are netstandard2.0 targeted, in ServiceHubCore they are 
             // .net targeted. We need to always pick the same set of assemblies regardless of who causes us to load. Because this code only
             // runs in a .net based host, it's safe to always choose the .net targeted ServiceHubCore versions.
-            if (!directory.AsSpan().TrimEnd('/', '\\').EndsWith(ServiceHubCoreFolderName, StringComparison.OrdinalIgnoreCase))
+            if (!directory.AsSpan().TrimEnd("/\\").EndsWith(ServiceHubCoreFolderName, StringComparison.OrdinalIgnoreCase))
```

Comment:
Oh no, please no. Use the separate invocation form.

Reply:
That's not equivalent though, is it? 

```csharp
var x = "abc/\\/";
x.TrimEnd("/\\")              // "abc"
x.TrimEnd("/").TrimEnd("\\")  // "abc/"
x.TrimEnd("\\").TrimEnd("/")  // "abc/\"
```

Reply:
There is no consistency in what the string overload here means, which is why string.Trim(ROS<char>) was pulled from 9 in RC2. I don't know where this string-accepting extension is defined, but it's ambiguous whether it means trim exactly the string provided, or trim any of the characters provided. In any case, a collection expression of both items should work, and should be non-ambiguous as to the meaning.

Reply:
I'm not entirely sure what your suggestion is here @333fred. Are you saying it should use a collection expr for the characters?

Reply:
> Are you saying it should use a collection expr for the characters?

Correct. Admittedly, part of this is just that the string version is making me queasy given the previous history around this method in .NET 9, but I do think we should opt for clear semantics, not ambiguous ones.

Reply:
@333fred responded to feedback

Reply:
Even better solution.